app/models/ApprovalResponse.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.{Json, OFormat}
+
+// TODO confirm what is returned as id is just the ocelot process id
+case class ApprovalResponse(id: String)
```

Comment:
I wonder if we should use SaveApprovalResponse here. In the future we will implement more related endpoints to change the status of a process awaiting approval. These endpoints will probably also require response models. As such I feel ApprovalResponse is a little too general.

Reply:
I'll leave this as is at present until we get more information on what the UI needs to send and receive during the approval process.  Once we have that, we can work out what responses we need.

Reply:
Ok